pkg/api/common_test.go

```diff
@@ -376,9 +379,10 @@ func setupHTTPServerWithCfgDb(t *testing.T, useFakeAccessControl, enableAccessCo
 		Live:               newTestLive(t),
 		QuotaService:       &quota.QuotaService{Cfg: cfg},
 		RouteRegister:      routeRegister,
-		SQLStore:           store,
 		searchUsersService: searchusers.ProvideUsersService(db, filters.ProvideOSSSearchUserFilter()),
-		dashboardService:   dashboardservice.ProvideDashboardService(dashboardsStore, nil),
+		SQLStore:           store,
+		dashboardService:   dashboardservice.ProvideDashboardService(dashboardsStore, nil, starsFake),
+		StarsService:       starsFake,
```

Comment:
Maybe `StarManager` and `starFake`? We should probably discuss in a group "service" vs "manager" naming.